test/lifecycle_module_test.dart

```diff
@@ -50,11 +52,11 @@ class TestLifecycleModule extends LifecycleModule {
   List<String> eventList;
   bool mockShouldUnload;
 
-  TestLifecycleModule()
+  TestLifecycleModule({String name})
       : managedDisposable = new Disposable(),
         managedStreamController = new StreamController<Null>(),
         managedStreamSubscription = new MockStreamSubscription(),
-        name = 'TestLifecycleModule' {
+        this.name = name {
```

Comment:
The test class overrides the name getter and provides a setter in the test implementation. This is not using the deprecated setter in the base implementation.